src/lib/pubkey/frodokem/frodo_types.h

```diff
@@ -0,0 +1,44 @@
+/*
+* FrodoKEM modes and constants
+*
+* The Fellowship of the FrodoKEM:
+* (C) 2023 Jack Lloyd
+*     2023 René Meusel, Amos Treiber - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_FRODOKEM_TYPES_H_
+#define BOTAN_FRODOKEM_TYPES_H_
+
+#include <botan/secmem.h>
+#include <botan/strong_type.h>
+
+#include <array>
+#include <vector>
+
+namespace Botan {
+
+using FrodoSeedA = Strong<std::vector<uint8_t>, struct FrodoSeedA_>;
+using FrodoSeedS = Strong<secure_vector<uint8_t>, struct FrodoSeedS_>;
+using FrodoSeedSE = Strong<secure_vector<uint8_t>, struct FrodoSeedSE_>;
+using FrodoSeedZ = Strong<std::vector<uint8_t>, struct FrodoSeedZ_>;
+using FrodoSampleR = Strong<secure_vector<uint8_t>, struct FrodoSampleR_>;
+using FrodoPublicKeyHash = Strong<std::vector<uint8_t>, struct FrodoPublicKeyHash_>;
+
+using FrodoPackedMatrix = Strong<std::vector<uint8_t>, struct FrodoPackedMatrix_>;
+using FrodoSerializedMatrix = Strong<secure_vector<uint8_t>, struct FrodoSerializedMatrix_>;
+
+using FrodoDomainSeparator = Strong<std::array<uint8_t, 1>, struct FrodoDoaminSeparator_>;
+
+/// u/u' in the spec
+using FrodoPlaintext = Strong<secure_vector<uint8_t>, struct FrodoPlaintext_>;
+
+using FrodoSalt = Strong<std::vector<uint8_t>, struct FrodoSalt_>;
+
+// TODO: Find a better name for this
+using FrodoK = Strong<secure_vector<uint8_t>, struct FrodoK_>;
```

Comment:
Maybe you can add some comments about the types (+ their respective symbol in the spec)